subsys/bluetooth/host/conn.c

```diff
@@ -1116,6 +1121,64 @@ uint8_t bt_conn_index(struct bt_conn *conn)
 	return (uint8_t)index;
 }
 
+
+#if defined(CONFIG_NET_BUF_LOG)
+struct net_buf *bt_conn_create_pdu_timeout_debug(struct net_buf_pool *pool,
+						 size_t reserve,
+						 k_timeout_t timeout,
+						 const char *func, int line)
+#else
+struct net_buf *bt_conn_create_pdu_timeout(struct net_buf_pool *pool,
+					   size_t reserve, k_timeout_t timeout)
+#endif
```

Comment:
This function was just moved

Reply:
Could you make moving of functions a separate commit? Makes it easier to view the change you made, not just for the review, but also in the history later.

Reply:
Sure

Reply:
@joerchan Split the PR into 3 commits
1) Move of create_pdu
2) Move and update for num_complete_packets
3) Remaining TX/RX issues found